src/components/NextButton/NextButton.tsx

```diff
@@ -1,5 +1,7 @@
 // SPDX-License-Identifier: MPL-2.0
-// Copyright (C) 2020 - 2022 Gemeente Amsterdam
+// Copyright (C) 2020 - 2023 Gemeente Amsterdam
+import type { BaseSyntheticEvent, ReactNode } from 'react'
```

Comment:
What is the reason you chose `BaseSyntheticEvent` and not just `React.MouseEvent` or something?

Reply:
That is the type of onClick in this IncidentNavigationProps, but the NextButton component is a generic component that can be used anywhere, so the onClick should be generic as well. Not sure what BaseSyntheticEvent adds to the type though. 
@OscarBakker can you explain why you used this type (since it it not used anywhere else) and why a simple MouseEvent does not suffice?